pup-0002.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Docstring Style Conventions
+author: Austin Macdonald
+created: 28-Mar-2017
+status: proposed
+---
+
+## Summary
+
+This PUP proposes the use of Google style docstrings for Pulp 3+.
+
+## Motivation
+
+Most current Pulp developers find Google Style docstrings to be more readable than the default
+Sphinx style docstrings.
+
+## Details
```

Comment:
I think this section can be eliminated and the 'Style Choices' section would become a two-pound indendend instead of three.